packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 40.10.0 - September 14, 2017
+
+### Added
+#### salesforcedx-vscode-apex
+
+* Go To Definition feature for Apex methods and constructors within the current file. ([PR #104](https://github.com/forcedotcom/salesforcedx-vscode/pull/104))
+#### salesforcedx-vscode-core
+
+* SFDX: View Local Changes and SFDX: View Changes in Default Scratch Org commands. ([PR #102](https://github.com/forcedotcom/salesforcedx-vscode/pull/102))
```

Comment:
Suggested version:
* New commands: SFDX: View Local Changes and SFDX: View Changes in Default Scratch Org. ([PR #102](https://github.com/forcedotcom/salesforcedx-vscode/pull/102))

Reply:
Cool. I wanted to add that but wasn't sure if that was redundant since it was under the Added section.

Reply:
Actually, let's use the format we used before and make a bullet point for each command, to make the change log easier to scan:
`#### salesforcedx-vscode-core

* SFDX: View Local Changes command ([PR #102](https://github.com/forcedotcom/salesforcedx-vscode/pull/102))
* SFDX: View Changes in Default Scratch Org command ([PR #102](https://github.com/forcedotcom/salesforcedx-vscode/pull/102))`